Sources/BottomSheet/BottomSheet.swift

```diff
@@ -61,15 +61,19 @@ public struct BottomSheet<Content: View>: View {
                         Spacer()
                     }
                 }
-                .frame(height: self.height - min(self.draggedOffset*2, 0))
+                .frame(height: heightFix(geometry: geometry) - min(self.draggedOffset*2, 0))
                 .background(self.contentBackgroundColor)
                 .cornerRadius(self.topBarCornerRadius, corners: [.topLeft, .topRight])
                 .animation(.interactiveSpring())
-                .offset(y: self.isPresented ? (geometry.size.height/2 - self.height/2 + geometry.safeAreaInsets.bottom + self.draggedOffset) : (geometry.size.height/2 + self.height/2 + geometry.safeAreaInsets.bottom))
+                .offset(y: self.isPresented ? (geometry.size.height/2 - heightFix(geometry: geometry)/2 + geometry.safeAreaInsets.bottom + self.draggedOffset) : (geometry.size.height/2 + heightFix(geometry: geometry)/2 + geometry.safeAreaInsets.bottom))
             }
         }
     }
     
+    fileprivate func heightFix(geometry: GeometryProxy) -> CGFloat {
```

Comment:
`heightFix` as func name seems bit vague for me. Will `func sheetHeight(in:)` be better?